module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -452,127 +451,128 @@ namespace module::behaviour::strategy {
+        bool feet_off_ground = !sensors.feet[BodySide::LEFT].down && !sensors.feet[BodySide::RIGHT].down;
+        return false && feet_off_ground && !is_getting_up && sensors.Htw(2, 2) < 0.92 && sensors.Htw(2, 2) > 0.88;
     }
 
     bool SoccerStrategy::penalised() const {
-        return selfPenalised;
+        return self_penalised;
     }
 
-    bool SoccerStrategy::ballDistance(const Ball& ball) {
+    bool SoccerStrategy::ball_distance(const Ball& ball) {
         return ball.position.norm() != 0.0;
     }
 
-    void SoccerStrategy::find(const std::vector<FieldTarget>& fieldObjects) {
+    void SoccerStrategy::find(const std::vector<FieldTarget>& field_objects) {
         // Create the soccer object priority pointer and initialise each value to 0.
-        auto soccerObjectPriority  = std::make_unique<SoccerObjectPriority>();
-        soccerObjectPriority->ball = 0;
-        soccerObjectPriority->goal = 0;
-        soccerObjectPriority->line = 0;
-        for (const auto& fieldObject : fieldObjects) {
-            switch (fieldObject.target.value) {
+        auto soccer_object_priority  = std::make_unique<SoccerObjectPriority>();
+        soccer_object_priority->ball = 0;
+        soccer_object_priority->goal = 0;
+        soccer_object_priority->line = 0;
+        for (const auto& field_object : field_objects) {
+            switch (field_object.target.value) {
                 case FieldTarget::Target::SELF: {
-                    soccerObjectPriority->goal        = 1;
-                    soccerObjectPriority->search_type = SearchType::GOAL_SEARCH;
+                    soccer_object_priority->goal        = 1;
+                    soccer_object_priority->search_type = SearchType::GOAL_SEARCH;
 
                     break;
                 }
                 case FieldTarget::Target::BALL: {
-                    soccerObjectPriority->ball        = 1;
-                    soccerObjectPriority->search_type = SearchType::LOST;
+                    soccer_object_priority->ball        = 1;
+                    soccer_object_priority->search_type = SearchType::LOST;
                     break;
                 }
                 default: throw std::runtime_error("Soccer strategy attempted to find a bad object");
             }
         }
-        emit(std::move(soccerObjectPriority));
+        emit(std::move(soccer_object_priority));
     }
 
-    Eigen::Vector2d SoccerStrategy::getKickPlan(const Field& field, const FieldDescription& fieldDescription) {
+    Eigen::Vector2d SoccerStrategy::get_kick_plan(const Field& field, const FieldDescription& field_description) {
         // Defines the box within in which the kick target is changed from the centre
         // of the oppposition goal to the perpendicular distance from the robot to the goal
 
-        const float maxKickRange =
+        const float max_kick_range =
             0.6;  // TODO(BehaviourTeam): make configurable, only want to change at the last kick to avoid smart goalies
```

Comment:
Can likely add a config for this in the yaml for the "default" value of max_kick_range, given the comment I'm assuming that this is due to be a value that changes over time depending on the game state, although that can be implemented at another time

Reply:
This functionality isn't used yet in our behavior due to localization being poor, I think we should come back to this once we try to implement it again.